packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -2762,7 +3036,9 @@ type WithControllerCallback<ReturnValue> = ({
   messenger: KeyringControllerMessenger;
 }) => Promise<ReturnValue> | ReturnValue;
 
-type WithControllerOptions = Partial<KeyringControllerOptions>;
+type WithControllerOptions = Partial<KeyringControllerOptions> & {
```

Comment:
Nit: Something I've started doing in other test files when it comes to `withController` (or other similar helpers, like `setupController`) is splitting off controller options from other kinds of options. So `withController` could take an options object, but the controller options could be bundled into an `options` object, and then any other setup configuration you want to make could be done in other options. Essentially that turns this type into:

``` typescript
type WithControllerOptions = {
  options: Partial<KeyringControllerOptions>;
  skipVaultCreation?: boolean;
}
```

I realize that this would require a change across the board, but something to think about perhaps for a future PR.

Reply:
Good idea, I do think it would improve readability! I will do that in another PR